src/main/java/org/broadinstitute/hellbender/tools/exome/AllelicCountCollection.java

```diff
@@ -33,10 +34,14 @@ public AllelicCountCollection() {
      * @param inputFile     file to read from
      */
     public AllelicCountCollection(final File inputFile) {
+        Utils.nonNull(inputFile);
+        Utils.regularReadableUserFile(inputFile);
         try (final TableReader<AllelicCount> reader = TableUtils.reader(inputFile,
                 (columns, formatExceptionFactory) -> {
-                    if (!columns.matchesExactly(AllelicCountTableColumns.COLUMN_NAME_ARRAY))
-                        throw formatExceptionFactory.apply("Bad header");
+                    if (!columns.containsAll(AllelicCountTableColumns.COLUMN_NAME_ARRAY)) {
```

Comment:
i agree (both about extracting the method and having a fixed exception).  will file an issue!